modules/ROOT/pages/kubernetes/operations/backup-restore.adoc

```diff
@@ -597,6 +612,142 @@ This pod performs a backup of one or multiple databases, a consistency check of
 . Monitor the backup pod logs using `kubectl logs pod/<neo4j-backup-pod-name>` to check the progress of the backup.
 . Check that the backup files and the consistency check reports have been uploaded to the cloud provider bucket or on-premises storage.
 
+[role=label--new-5.21]
+[[kubernetes-neo4j-backup-aggregate]]
+== Aggregate a database backup chain
+
+The aggregate backup command turns a backup chain into a single backup file.
+This is useful when you have a backup chain that you want to restore to a different cluster, or when you want to archive a backup chain.
+For more information on the benefits of the aggregate backup chain operation, its syntax and available options, see xref:backup-restore/aggregate.adoc[Aggregate a database backup chain].
+
+[NOTE]
+====
+The neo4j-admin Helm chart supports aggregating a backup chain stored in an AWS S3 bucket or a local mount.
+If enabled, normal backup will not be done, only aggregate backup.
+====
+
+. To aggregate a backup chain stored in an AWS S3 bucket or a local mount, you need to provide the following information in your _backup-values.yaml_ file:
++
+[.tabbed-example]
+=====
+[.include-with-aws]
+======
+
+If your backup chain is stored on AWS S3, you need to set cloudProvider to `aws` and use either `creds` or `serviceAccount` to connect to your AWS S3 bucket.
+For example:
+
+.Connect to your AWS S3 bucket using the `awscreds` secret
+[source, yaml, role='noheader']
+----
+neo4j:
+  image: "neo4j/helm-charts-backup"
+  imageTag: "5.21.0"
+  jobSchedule: "* * * * *"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 1
+  backoffLimit: 3
+
+backup:
+
+  cloudProvider: "aws"
+  secretName: "awscreds"
+  secretKeyName: "credentials"
+
+  aggregate:
+    enabled: true
+    verbose: false
+    keepOldBackup: false
+    parallelRecovery: false
+    fromPath: "s3://bucket1/bucket2"
+    # Database name to aggregate. Can contain * and ? for globbing.
+    database: "neo4j"
+
+resources:
+  requests:
+    ephemeralStorage: "4Gi"
+  limits:
+    ephemeralStorage: "5Gi"
+----
+
+.Connect to your AWS S3 bucket using `serviceAccount`
+[source, yaml, role='noheader']
+----
+neo4j:
+  image: "neo4j/helm-charts-backup"
+  imageTag: "5.21.0"
+  jobSchedule: "* * * * *"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 1
+  backoffLimit: 3
+
+backup:
+
+    cloudProvider: "aws"
+
+    aggregate:
+      enabled: true
+      verbose: false
+      keepOldBackup: false
+      parallelRecovery: false
+      fromPath: "s3://bucket1/bucket2"
+      # Database name to aggregate. Can contain * and ? for globbing.
+      database: "neo4j"
+
+#The service account must already exist and have the necessary permissions to access and manage your S3 bucket. See https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies.html for more information.
+serviceAccountName: "my-service-account"
```

Comment:
we should add the permissions required in the policy for the aggregate backup to work. Have shared the required permissions in the demo video. either we add those here or add them in the aggregate backup documentation and reference it here